### PR TITLE
maximum segments reached and Select all bugfix

### PR DESCRIPTION
**Pull request by blazoncek**
Fixes "Maximum segments reached" text and corrects behaviour of Select all checkbox

### CODE REVIEW

**Comment by Aircoookie** [comment]
Confirmed working as should (including unselect)!

**Review by Aircoookie — APPROVED** [review]

File: wled00/data/index.js [hunk 1]
```diff
@@ -1694,15 +1689,15 @@ function makeSeg()
...
 
-function resetUtil()
+function resetUtil(off=false)
 {
-//	gId('segutil').innerHTML = '<button class="btn btn-s" onclick="makeSeg()"><i class="icons btn-icon">&#xe18a;</i>segment</button>';
-	gId('segutil').innerHTML = '<div class="seg btn btn-s" style="border-radius:24px;padding:0;">'
+	gId('segutil').innerHTML = `<div class="seg btn btn-s ${off?'off':''}" style="border-radius:24px;padding:0;">`
 	+ '<label class="check schkl"><input type="checkbox" id="selall" onchange="selSegAll(this)"><span class="checkmark"></span></label>'
-	+ '<div class="segname" onclick="makeSeg()"><i class="icons btn-icon">&#xe18a;</i>Add segment</div></div>';
+	+ `<div class="segname" ${off?'':'onclick="makeSeg()"'}><i class="icons btn-icon">&#xe18a;</i>Add segment</div></div>`;
```
Comment (softhack007) [inline wled00/data/index.js:1696]: Is is correct to use a "back-tick" here? Just curious, as I'm not so much experienced with JS.

**Review by softhack007 — APPROVED** [review]
Nothing from my side, just curious question about using "back-tick" instead of single quotes.

File: wled00/data/index.js (hunk 1 above)
Comment (blazoncek) [reply wled00/data/index.js:1696]: Back-tick (as I learned during my WLED tenure) is a special string marker. Anything within back-ticks is paresed in a similar way as PHP parses strtings. `${...}` is the parsing trick. Within it, regular JS can be embedded.

**Pull request merged**